### ISSUE TITLE
Windows 10 19041.2908 need support

### ISSUE THREAD

**Issue by qaz1qazlol2** [stascorp/rdpwrap#2171]

**Comment by sashaqwert** [comment]
Please, attach your RDPConf screenshot and `termsrv.dll`.

10.0.19041.2908 (keyword for search)

**Comment by qaz1qazlol2** [comment]
![1](https://user-images.githubusercontent.com/115929847/231985998-e3ccd7a9-9668-49b5-b8f7-e587f0462579.png)
[termsrv.zip](https://github.com/stascorp/rdpwrap/files/11230486/termsrv.zip)

**Comment by qaz1qazlol2** [comment]
fixed

**Comment by sebaxakerhtc** [comment]
> fixed

Test if it works normally. Because there's no pdb file and i copied the offsets from a previous version, but checked they are the same

**Comment by qaz1qazlol2** [comment]
![SharedScreenshot](https://user-images.githubusercontent.com/115929847/232067396-fb8780fb-b626-41e8-956e-bd623bef24e0.jpg)
![2](https://user-images.githubusercontent.com/115929847/232067421-6eb8eb89-28fc-4342-acc0-ed5db9a40b80.png)
well it actually works

**Comment by qaz1qazlol2** [comment]
Should I close this issue?

**Comment by sebaxakerhtc** [comment]
> Should I close this issue?

No. There's a rule to leave opened 1 issue per termsrv.dll

**Comment by qaz1qazlol2** [comment]
ok,thanks

**Comment by ReaperAlpha** [comment]
Why are there no new builds? I looked and the latest build is from 2017... why haven't we updated the builds? how do I get this update without the build? Are we really expecting everyone to compile their own build each time?

**Comment by sebaxakerhtc** [comment]
> Why are there no new builds? I looked and the latest build is from 2017... why haven't we updated the builds? how do I get this update without the build? Are we really expecting everyone to compile their own build each time?

Official repo is abandoned. So, use my fork or my ini

**Comment by ReaperAlpha** [comment]
> > Why are there no new builds? I looked and the latest build is from 2017... why haven't we updated the builds? how do I get this update without the build? Are we really expecting everyone to compile their own build each time?
> 
> Official repo is abandoned. So, use my fork or my ini

I found an updater that made my functional in a thread below. Is this updater just pushing your Ini?
https://github.com/Octanium91/rdpwupDownloads/blob/master/rdpwup_8.zip

**Comment by sebaxakerhtc** [comment]
> > > Why are there no new builds? I looked and the latest build is from 2017... why haven't we updated the builds? how do I get this update without the build? Are we really expecting everyone to compile their own build each time?
> > 
> > 
> > Official repo is abandoned. So, use my fork or my ini
> 
> I found an updater that made my functional in a thread below. Is this updater just pushing your Ini? https://github.com/Octanium91/rdpwupDownloads/blob/master/rdpwup_8.zip

Yeah, I think so. Like @affinityv or @asmtron
The difference is @affinityv say thanks 😁

**Comment by ReaperAlpha** [comment]
> > > > Why are there no new builds? I looked and the latest build is from 2017... why haven't we updated the builds? how do I get this update without the build? Are we really expecting everyone to compile their own build each time?
> > > 
> > > 
> > > Official repo is abandoned. So, use my fork or my ini
> > 
> > 
> > I found an updater that made my functional in a thread below. Is this updater just pushing your Ini? https://github.com/Octanium91/rdpwupDownloads/blob/master/rdpwup_8.zip
> 
> Yeah, I think so. Like @affinityv or @asmtron The difference is @affinityv say thanks 😁

Thank you for the clarity here, I wonder if we could put a link to your info near the top so people dont have to keep asking these silly questions. At least I feel silly asking for clarity when seemingly others are making it work XD

Seems to be working Thank you so much. I'm going to try and uninstall this old version and install yours

**Comment by sebaxakerhtc** [comment]
> I wonder if we could put a link to your info near the top

No, we can't. Stas M approved my mod, but don't want to place links to it. Maybe one day he will return to the project...

**Comment by sashaqwert** [comment]
Offsets

| Architecture | Commit |
|--------------|---------|
| x64               | https://github.com/sebaxakerhtc/rdpwrap.ini/commit/357771ebd83344c21d1a814b8aa717c20704baa5 |
| x86               | https://github.com/sebaxakerhtc/rdpwrap.ini/commit/bef52d35011936a80dca6d9281c1d406ff3b18c9    |

**Comment by MrGamer98** [comment]
> _No description provided._

how to get free vps/rdp??

**Comment by affinityv** [comment]
> > > > Why are there no new builds? I looked and the latest build is from 2017... why haven't we updated the builds? how do I get this update without the build? Are we really expecting everyone to compile their own build each time?
> > > 
> > > 
> > > Official repo is abandoned. So, use my fork or my ini
> > 
> > 
> > I found an updater that made my functional in a thread below. Is this updater just pushing your Ini? https://github.com/Octanium91/rdpwupDownloads/blob/master/rdpwup_8.zip
> 
> Yeah, I think so. Like @affinityv or @asmtron The difference is @affinityv say thanks grin

@sebaxakerhtc -- your work is outstanding, EVERYBODY should be thanking you!
